linode_api4/objects/networking.py

```diff
@@ -9,20 +9,56 @@ class IPv6Pool(Base):
     properties = {
         "range": Property(identifier=True),
         "region": Property(slug_relationship=Region, filterable=True),
+        "prefix": Property(),
+        "route_target": Property(),
     }
 
+    def _api_get(self):
+        """
+        A helper method to GET this object from the server
+        """
+        pools = self._client.networking.ipv6_pools()
+        pool = [p for p in pools if p.range == self.range][0]
+
+        if pool is not None:
+            self._populate(pool._raw_json)
+        else:
+            raise LookupError("Could not find IPv6 pool with proviced range.")
+
 
 class IPv6Range(Base):
-    api_endpoint = "/networking/ipv6/ranges/{}"
+    api_endpoint = "/networking/ipv6/ranges/{range}"
     id_attribute = "range"
 
     properties = {
         "range": Property(identifier=True),
         "region": Property(slug_relationship=Region, filterable=True),
+        "prefix": Property(),
+        "route_target": Property(),
     }
 
+    def ip_ranges_list(self):
```

Comment:
Should we use `PaginatedList` to return the list of ipv6 ranges? It seems able to be paged on the api doc https://www.linode.com/docs/api/networking/#ipv6-ranges-list

Reply:
I'm honestly note sure 😆. For other list endpoints I have been returning a list of the objects returned from the API directly without using a `PaginatedList` but I'm not sure that is the best way to do it. Does anyone else have any thoughts?

Reply:
+1. I'm asking this because I'm implementing object_storage_buckets_list, and it also can be paged in the doc lol. But looking around in our code base, I only find one example using `PaginatedList` to return paged list: https://github.com/linode/linode_api4-python/blob/1456769ce88c5fc1e963dd924bdf5fc8a1f1c7c0/linode_api4/objects/tag.py#L64
And there isn't a test case for this get list method. Not sure if it works as we expect or not

Reply:
Hmm, yeah I've noticed that there are some gaps in this repo when it comes to testing. I would say that if most list methods do not use a `PaginatedList` then it would probably be better to follow suit but I'm not sure.

Reply:
I think PaginatedList would be better in this case. Does `self.client._get_and_filter` do the trick?